scripts/build/cc/100-gcc.sh

```diff
@@ -403,10 +403,12 @@ do_gcc_core_backend() {
         extra_config+=("--with-system-zlib")
     fi
 
-    # Some versions of gcc have a deffective --enable-multilib.
+    # Some versions of gcc have a defective --enable-multilib.
     # Since that's the default, only pass --disable-multilib.
     if [ "${CT_MULTILIB}" != "y" ]; then
         extra_config+=("--disable-multilib")
+    else
+        extra_config+=("--enable-multiarch")
```

Comment:
Mutlilib makes GCC aware of different library locations/sysroots for different sets of flags. Multiarch adds to that an ability to report the configuration tuple corresponding to these flags. So, multiarch only makes sense if you have multilib. The --enable-multiarch option is actually a tri-state, it can be enabled/disabled/left as auto. The default is 'auto', and the 'auto' logic enables it if it finds multiple subdirectories in ${libdir} with C runtime objects (crt1.o, etc). So, in pass-2/final the multiarch was enabled already if multilib was enabled. The problem was with core pass-1: we don't have the CRT installed yet, so multiarch support was not detected. Effectively, this change applies the status quo configuration in pass-2/final to pass-1 as well.

And, as the comment above this conditional states, --enable-multilib is the default, so we don't need to explicitly pass that.